msmbuilder/commands/featurizer.py

```diff
@@ -199,3 +204,38 @@ def _solvent_indices_type(self, fn):
 
     def _solute_indices_type(self, fn):
         return np.loadtxt(fn, dtype=int, ndmin=1)
+
+
+class VonMisesFeaturizerCommand(FeaturizerCommand):
+    _concrete = True
+    klass = VonMisesFeaturizer
+
+
+class FunctionFeaturizerCommand(FeaturizerCommand):
```

Comment:
Yeah, this needs some sort of `_def_function` or if it is unsuitable for command line then either (a) a way to classify whether functions are suitable or unsuitable from the outset or (b) more hacking of the test